migration/sql-files/048-categories.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text
+);
+
+ALTER TABLE work_item_types ADD COLUMN categories_id uuid;
```

Comment:
The field should be singular `category_id` instead of `categories_id`.

We might need to set a default value and `not null` constraints for the `category_id`.  Something like we did with the `space_id` in the work_items `ALTER TABLE work_item_types ADD space_id uuid DEFAULT '{{index . 0}}' NOT NULL;` https://github.com/almighty/almighty-core/blob/master/migration/migration.go#L211.